Phonebook.Frontend/src/migration/migrations/3/migration3.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { migration3 } from './migration3';
+
+const APPSTATE_KEY: string = 'appstate';
+
+describe('Migration: 3', () => {
+  it(migration3.name, () => {
+    localStorage.setItem(
+      APPSTATE_KEY,
+      `{"serviceWorkerNotificationDisplayed":false,"version":"1.0.0","displayedNotificationVersion":1,"sendFeedback":false}`
+    );
+    migration3.script();
+    expect(localStorage.getItem(APPSTATE_KEY)).toEqual(
+      `{"serviceWorkerNotificationDisplayed":false,"version":"1.0.0","displayedNotificationVersion":1,"sendFeedback":false,"activeTheme":"magenta_light_theme"}`
+    );
+  });
+
+  it('shoud do no harm if data is already migrated', () => {
+    const alreadyMigratedString = `{"serviceWorkerNotificationDisplayed":false,"version":"1.0.0","displayedNotificationVersion":1,"sendFeedback":false,"activeTheme":"magenta_light_theme"}`;
```

Comment:
the migration sets `activeTheme` to `magenta_light_theme` anyway. It would be better to set the previous value to something else

Reply:
done